examples/juniper/cptx-ixia/cptx-ixia.pb.txt

```diff
@@ -0,0 +1,110 @@
+name: "ixia-c"
+nodes: {
+    name: "otg"
+    vendor: KEYSIGHT
+    version: "0.0.1-3423-local"
+    services: {
+        key: 443
+        value: {
+            name: "ixia-c"
+            inside: 443
+            outside: 0
+        }
+    }
+    services: {
+        key: 40051
+        value: {
+            name: "grpc"
+            inside: 40051
+            outside: 0
+        }
+    }
+    services: {
+        key: 50051
+        value: {
+            name: "gnmi"
+            inside: 50051
+            outside: 0
+        }
+    }
+}
+
+nodes: {
+    name: "cptx1"
+    vendor: JUNIPER
+    model: "cptx"
+    os: "evo"
+    config: {
+      image: "cptx:test"
+      config_path: "/home/evo/configdisk"
+      config_file: "juniper.conf"
+      file: "juniper.config"
+      cert: {
+            self_signed: {
+                cert_name: "ca-ipsec",
+                key_name: "N/A",
+                key_size: 4096,
+            }
+        }
+    }
+    interfaces: {
+        key: "eth12"
+        value: {
+          name: "et-0/0/1:0"
+        }
+    }
+    interfaces: {
+        key: "eth20"
+        value: {
+          name: "et-0/0/2:0"
+        }
+    }
+    interfaces: {
+        key: "eth28"
+        value: {
+          name: "et-0/0/3:0"
+        }
+    }
+   services:{
+        key: 22
+        value: {
+            name: "ssh"
+            inside: 22
+        }
+    }
+    services:{
```

Comment:
can you add the other openconfig services (if they are not supported yet please leave a comment with a TODO)?

also please use the standard outside ports, see the below example:

```
  services:{
        key: 9337
        value: {
            name: "gnoi"
            inside: 57400
            outside: 9337
        }
    }
    services:{
        key: 9339
        value: {
            name: "gnmi"
            inside: 57400
            outside: 9339
        }
    }
    services:{
        key: 9340
        value: {
            name: "gribi"
            inside: 57401
            outside: 9340
        }
    }
    services:{
        key: 9559
        value: {
            name: "p4rt"
            inside: 9559
            outside: 9559
        }
    }
```

Reply:
Done, added a README.md too.

Reply:
Also, note that the cPTX config will not need additional changes to enable other previously missing services.